Components/grade-table.js

```diff
@@ -0,0 +1,20 @@
+class GradeTable {
+    constructor(tableElement) {
+        this.tableElement = tableElement;
+    }
+    updateGrades(grades) {
+        var treeBody = this.tableElement.querySelector("tbody");
+            treeBody.innerHTML = "";
+                for(var i = 0; i < grades.length; i++) {
+                    var newTreeRow = document.createElement("tr");
+                    var studentName = document.createElement("td");
+                    var studentCourse = document.createElement("td");
+                    var studentGrade = document.createElement("td");
+                        studentName.textContent = grades[i].name;
```

Comment:
Your formatting started to get away from you here, please bring this section back to proper indentation 👍